test.py

```diff
@@ -0,0 +1,11 @@
+en1_moves_dict = {'bp': 0, 'category': 'Status', 'type': 'Normal'}
```

Comment:
Do we still need this test? If not let's delete. If we want to keep it let's move it to a folder called "tests".